app/src/main/java/com/example/KnowYouLimit/service/DrinksService.kt

```diff
@@ -0,0 +1,10 @@
+package com.example.KnowYouLimit.service
+
+import androidx.lifecycle.MutableLiveData
+import com.example.KnowYouLimit.dto.drinks
+
+class DrinksService {
+    fun fetchDrinks(name: String): MutableLiveData<ArrayList<drinks>> {
+        return MutableLiveData<ArrayList<drinks>>()
+    }
+}
```

Comment:
Simple drink service addition explained very nicely in Professor Jones module 4 videos.